File: android_app_v2/app/src/main/java/com/airec/bledemo/designsystem/components/SectionLabel.kt
package com.airec.bledemo.designsystem.components

import androidx.compose.foundation.layout.Arrangement
import androidx.compose.foundation.layout.Row
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.layout.size
import androidx.compose.material3.Icon
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.vector.ImageVector
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp
import com.airec.bledemo.designsystem.MeiliIcons
import com.airec.bledemo.designsystem.MeiliPalette
import com.airec.bledemo.designsystem.MeiliTheme

/**
 * 区块小标题 [SectionLabel]，还原 warm_2.html 的 .section-lbl：
 * 11.5sp / 800 / ink-2 字、0.5 字距、可选前置鼠尾草色图标（15dp）。
 *
 * 用于「陪伴设备」「本次将分析的陪伴」「累积标签」等分组标题。
 *
 * @param text 标题文字
 * @param icon 可选前置线性图标（默认鼠尾草深色，呼应原型 .section-lbl .ic）
 */
@Composable
fun SectionLabel(
    text: String,
    modifier: Modifier = Modifier,
    icon: ImageVector? = null,
) {
    Row(
        modifier = modifier.padding(horizontal = 2.dp, vertical = 0.dp),
        horizontalArrangement = Arrangement.spacedBy(6.dp),
        verticalAlignment = Alignment.CenterVertically,
    ) {
        if (icon != null) {
            Icon(
                icon,
                contentDescription = null,
                tint = MeiliPalette.SageDeep,
                modifier = Modifier.size(15.dp),
            )
        }
        Text(
            text = text,
            style = MaterialTheme.typography.labelMedium.copy(
                fontSize = 11.5f.sp,
                fontWeight = FontWeight.ExtraBold,
                letterSpacing = 0.5.sp,
            ),
            color = MeiliPalette.Ink2,
        )
    }
}

@Preview(showBackground = true, backgroundColor = 0xFFFFFCF8, widthDp = 360)
@Composable
private fun SectionLabelPreview() {
    MeiliTheme {
        androidx.compose.foundation.layout.Column(
            modifier = Modifier.padding(18.dp),
            verticalArrangement = Arrangement.spacedBy(12.dp),
        ) {
            SectionLabel("陪伴设备", icon = MeiliIcons.Wifi)
            SectionLabel("本次将分析的陪伴（已绑定）", icon = MeiliIcons.Link)
            SectionLabel("累积标签 · 画像积累", icon = MeiliIcons.Star)
        }
    }
}

File: ios_app/MeiliCompanion/Sources/DesignSystem/Components/SectionLabel.swift
import SwiftUI

/// 区块小标题。android 端对应 `components/SectionLabel.kt`,还原 .section-lbl:
/// 11.5 / 800 / ink-2 字、0.5 字距、可选前置鼠尾草深色图标(15)。
/// 用于「陪伴设备」「本次将分析的陪伴」「累积标签」等分组标题。
struct SectionLabel: View {
    let text: String
    var icon: MeiliGlyph? = nil

    init(_ text: String, icon: MeiliGlyph? = nil) {
        self.text = text
        self.icon = icon
    }

    var body: some View {
        HStack(spacing: 6) {
            if let icon {
                MeiliIcon(icon, size: 15)
                    .foregroundStyle(MeiliColor.sageDeep)
            }
            Text(text)
                .font(.sz(11.5, weight: .heavy))
                .tracking(0.5)
                .foregroundStyle(MeiliColor.ink2)
        }
        .padding(.horizontal, 2)
    }
}
